flink-libraries/flink-table/pom.xml

```diff
@@ -148,80 +148,22 @@ under the License.
 						</goals>
 					</execution>
 				</executions>
-				<configuration>
-					<jvmArgs>
-						<jvmArg>-Xms128m</jvmArg>
-						<jvmArg>-Xmx512m</jvmArg>
-					</jvmArgs>
-					<compilerPlugins combine.children="append">
-						<compilerPlugin>
-							<groupId>org.scalamacros</groupId>
-							<artifactId>paradise_${scala.version}</artifactId>
-							<version>${scala.macros.version}</version>
-						</compilerPlugin>
```

Comment:
Are the macros only needed in flink-scala? I thought they are required for the type extraction using macros?